test/functional/fixtures/api/es-next/eval/test.js

```diff
@@ -1,10 +1,10 @@
 const expect         = require('chai').expect;
-const parseUserAgent = require('useragent').parse;
+const parseUserAgent = require('../../../../../../lib/utils/parse-user-agent');
 const config         = require('../../../../config');
 
 describe('[API] t.eval', function () {
     it('Should execute an anonymous client function', function () {
-        function assertUA (errs, alias, expected) {
+        function assertUA (errs, alias) {
```

Comment:
Looks like a duplicate of the `assertUA` from

https://github.com/DevExpress/testcafe/pull/4349/files#diff-bbc20d0b3be72d8d04b2a53d69e0dcebR7

Can we extract it to a module?